sdk/spring-sdk/src/test/scala/kalix/springsdk/impl/ActionDescriptorFactorySpec.scala

```diff
@@ -198,6 +202,42 @@ class ActionDescriptorFactorySpec extends AnyWordSpec with ComponentDescriptorSu
         assertRequestFieldJavaType(method, "one", JavaType.STRING)
       }
     }
+    "generate mappint with Event Sourced Entity Subscription annotation" in {
+      assertDescriptor[SubscribeToEventSourcedEntityAction] { desc =>
+        val methodDescriptorOne = desc.serviceDescriptor.findMethodByName("MethodOne")
+        methodDescriptorOne.isServerStreaming shouldBe false
+        methodDescriptorOne.isClientStreaming shouldBe false
+
+        val methodOne = desc.methods("MethodOne")
+        methodOne.requestMessageDescriptor.getFullName shouldBe JavaPbAny.getDescriptor.getFullName
+
+        val eventSourceOne = findKalixMethodOptions(desc, "MethodOne").getEventing.getIn
+        eventSourceOne.getEventSourcedEntity shouldBe "counter"
+
+        val ruleOne = findHttpRule(desc, "MethodOne")
+        ruleOne.getPost shouldBe "/kalix.springsdk.testmodels.subscriptions.PubSubTestModels.SubscribeToEventSourcedEntityAction/MethodOne"
+
+        val methodDescriptorTwo = desc.serviceDescriptor.findMethodByName("MethodTwo")
+        methodDescriptorTwo.isServerStreaming shouldBe false
+        methodDescriptorTwo.isClientStreaming shouldBe false
+
+        val methodTwo = desc.methods("MethodTwo")
+        methodTwo.requestMessageDescriptor.getFullName shouldBe JavaPbAny.getDescriptor.getFullName
+
+        val eventSourceTwo = findKalixMethodOptions(desc, "MethodTwo").getEventing.getIn
+        eventSourceTwo.getEventSourcedEntity shouldBe "counter"
+
+        val ruleTwo = findHttpRule(desc, "MethodTwo")
+        ruleTwo.getPost shouldBe "/kalix.springsdk.testmodels.subscriptions.PubSubTestModels.SubscribeToEventSourcedEntityAction/MethodTwo"
+
+      }
+    }
+
+    "fail if has both Event Sourced Entity Subscription and REST annotations" in {
```

Comment:
I assumed we don't want the users to add REST annotations themselves in combination with the @Subscribe.EventSourcedEntity. I think that this is because we already provide such an endpoint with @Subscribe.EventSourcedEntity but I think there is more to it.

Reply:
Correct. 

And on top of this, I think that they are automatically blocked by the ACL. They are meant for internal use.